tests/sles4sap/saptune/mr_test.pm

```diff
@@ -36,7 +37,12 @@ sub setup {
     # saptune is not installed by default on SLES4SAP 12 on ppc64le and in textmode profile
     zypper_call "-n in saptune" if ((is_ppc64le() and is_sle('<15')) or check_var('DESKTOP', 'textmode'));
     # Install mr_test dependencies
-    zypper_call "-n in python3-rpm";
+    # 'zypper_call "-n in python3-rpm"' returns error message:
+    #   "There are running programs which still use files and libraries deleted or updated by recent upgrades.
+    #   They should be restarted to benefit from the latest updates.
+    #   Run 'zypper ps -s' to list these programs."
+    # Then choosing 'script_run' instead.
+    script_run "zypper -n in python3-rpm";
```

Comment:
If you know the error code of `zypper` when installing `python3-rpm` you can perhaps use `zypper_call('in python3-rpm', exitcode => [0, $expected_retval])`.

Reply:
Done. Good to know this workaround. Thanks.